extension.driver.php

```diff
@@ -253,7 +253,7 @@ public function dataSourcePostExecute($context)
 
     // datasource output entries
 
-    private function findEntries(XMLElement $xml)
+    private function findEntries(XMLElement $xml, $entries_name = 'entry')
```

Comment:
I think a variable named `$node_name` would make more sense than `$entries_name` as we're trying to process both `entry`- and `item`-nodes. Could you change this?